qemu/tests/chardev_acpi.cfg

```diff
@@ -0,0 +1,5 @@
+- chardev_acpi:
+    only Linux 
+    no q35
+    type = chardev_acpi
+    chardev_backend_serial0 = tty
```

Comment:
If it is no must, don't use the default serial of our framework,  add a new serial for feature testing.